.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,79 @@
+name: Continuous integration
+
+on:
+  push:
+    branches:
+      - master
+      - release-**
+# TODO just for testing
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_REF: ${{ github.ref }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      TX_TOKEN: ${{ secrets.TX_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Install requirements
+        run: |
+          sudo apt update && sudo apt install texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended
+          mkdir -p $HOME/texlive && curl -L https://github.com/urdh/texlive-blob/releases/download/20200421/texlive.tar.xz | tar -JxC $HOME/texlive
+          pip install -r requirements.txt
+          PATH=$HOME/texlive/bin/x86_64-linux:$PATH
+          tlmgr option repository ctan
+          tlmgr update --verify-repo=none --self
+          tlmgr install combelow
+          tlmgr install babel-french
+          tlmgr install babel-german
+          tlmgr install babel-romanian
+          tlmgr install babel-italian
+          tlmgr install babel-spanish
+
+      - name: Release
+        run: |
+          git config --global user.email "info@opengis.ch";
+          git config --global user.name "opengisch";
+          git config --global push.default simple;
+
+      - name: Translations push
+        run: |
+          make gettext
+          tx push -s
+
+      - name: Translations push
+        run: |
+          set -e
+
+          make clean
+          if ${GITHUB_REF##*/} = "master";
```

Comment:
is this the right condition (the *secure* master branch term below)?

Reply:
Copy paste from `travis.yml`

Reply:
I guess we only want to run transifex on master, right? Then I think this is the right condition and the message needs to be updated?

Reply:
Yes, but the condition is for "secrets" related code, so I guess it shouldn't only be triggered by master?

Reply:
After second thought I think there is no need for the if condition at all. We can trigger the build, only we should filter whether the changes are commited to QField docs website.